packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/react",
-  "version": "2.0.4",
+  "version": "2.0.4-0.0.1",
```

Comment:
Apologies on this but you can revert to `2.0.5` here
We have temporarily moved away from feature branch versioning while we get out staging env build flow figured out.
Sorry about the requested change and revert.